.github/workflows/build.yaml

```diff
@@ -47,6 +45,24 @@ jobs:
           cmake3 -DCFL_DEV=on .
           make
 
+  build-debian:
+    name: Debian Buster build to confirm no issues once used downstream
+    runs-on: ubuntu-latest
+    container: debian:buster
+    steps:
+      - name: Set up base image dependencies
+        run: |
+          apt-get update
+          apt-get install -y build-essential cmake make
+        shell: bash
+
+      - uses: actions/checkout@v3
+
+      - name: Run compilation
+        run: |
+          cmake -DCFL_DEV=on .
+          make
```

Comment:
Does this run in `build/`? 
Could you use `cmake --build .` instead of `make`?

Reply:
Yeah can do, I just left it as it was before in the other builds.